README.md

```diff
@@ -34,23 +34,38 @@ go install github.com/coreruleset/go-ftw@latest
 
 ## Example Usage
 
-To run tests you need:
-1. a WAF (doh!)
-2. a file where the waf stores the logs
-3. a config file, or environment variables, with the information to get the logs and how to parse them (I might embed this for the most commonly used, like Apache/NGiNX), and additional options described next.
+The go-ftw is designed to run Web Application Firewall (WAF) unit tests. The primary focus is the [OWASP ModSecurity Core Rule Set](https://github.com/coreruleset/coreruleset).
+
+In order to run the tests, you need to prepare the following:
+
+1. Active WAF
+2. Log where the WAF writes the alert messages
+3. go-ftw config file `.ftw.yaml` in the local folder or in your home folder (see section "YAML Config file" for more information.
+4. At least one unit test in (go)-ftw's yaml format.
 
 ### YAML Config file
 
-With a configuration file you can alter the test results, set paths for your environment, or enable features. The config file has four basic values:
+With the configuration set paths for your environment, enable and disabled features and you can also use it to alter the test results.
+
+The config file has four basic values:
+
+* `logfile` : path to WAF log with alert messages, relative of absolute
+* `logmarkerheadername` : name of a HTTP header used for marking log messages, usually `X-CRS-TEST` (see section "How log parsing works" below)
+* `testoverride` : a list of things to override (see "Overriding tests" below)>
+* `mode` : "default" or "cloud" (only change it if you need "cloud")
+```
+
+__Example__:
 
 ```yaml
-logfile: <the relative path to the WAF logfile>
-logmarkerheadername: <a header name used for log parsing (see "How log parsing works" below)>
-testoverride: <a list of things to override (see "Overriding tests" below)>
-mode: "default" or "cloud" (only change it if you need "cloud")
+logfile: /apache/logs/error.log
+logmarkerheadername: X-CRS-TEST
+testoverride:
+mode: "default"
```

Comment:
Maybe we want to add more examples:
- this has mostly everything set, with the logfile being absolute
- add a relative logfile example
- add a minimal config: the bare minimum needed is the `logfile`.